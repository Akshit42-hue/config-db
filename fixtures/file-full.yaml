file:
  - type: $.InstanceType
    id: $.InstanceId
    full: true
    paths:
      - fixtures/config_change.json
