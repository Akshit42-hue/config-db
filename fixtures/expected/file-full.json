[
  {
    "last_modified": "0001-01-01T00:00:00Z",
    "type": "instance_type",
    "name": "instance_id_1",
    "id": "instance_id_1",
    "source": "/fixtures/config_change.json",
    "config": {
      "InstanceId": "instance_id_1",
      "InstanceType": "instance_type",
      "change": {
        "action": "speak",
        "summary": "speaking right now",
        "unrelated": 123,
        "external_id": "incident_commander",
        "external_type": "incident_commander"
      }
    },
    "changes": [
      {
        "action": "speak",
        "summary": "speaking right now",
        "external_id": "incident_commander",
        "external_type": "incident_commander"
      }
    ]
  }
]
